apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -0,0 +1,183 @@
+import * as React from 'react';
+// import { Notification } from
+import { View, Text } from 'react-native';
+import { Test, TestSection, PlatformStatus } from '../Test';
+// import { ColorValue } from 'react-native';
+
+
+const PrimaryToastTest: React.FunctionComponent = () => {
```

Comment:
Some notes:

1) "Toast" is the Android name for a notification: https://material.io/archive/guidelines/components/snackbars-toasts.html#snackbars-toasts-usage . I think they popularized the concept, so the name gets used in a lot of places (including our own designs). However, the right "FluentUI" name for this component is "Notification". That's also reflected in the name of the FluentUI Apple control. Could we change the name to "Notification"


2) I like that you have separate tests for each variant (Primary, Neutral, etc...)  of Notification! There's a lot of common code between them. Could we separate out the common code into one function component called `Notification`? Then `Notification` could takes one prop called "variant" and we could choose the styles / colors based on that. We could also give it more props for the strings inside each Text component on lines 37-38.